src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -46,20 +46,20 @@
     :render-fn                         render-fn}])
 
 (defn segments-community-lists [selected-tab]
-  (let [communities (rf/sub [:communities/community-ids])
+  (let [communities-ids (rf/sub [:communities/community-ids-by-user-involvement])
```

Comment:
Whenever I read `<something>-ids`, I think it's a collection (it's very conventional), but in reality it's a map indexed by the user "involvement". I would suggest renaming this binding to something like `ids-by-user-involvement`.